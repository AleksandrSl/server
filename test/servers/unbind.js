#!/usr/bin/env node

'use strict'

const Server = require('../../server')

const app = new Server({
  subprotocol: '1.0.0',
  supports: '1.x'
})
app.nodeId = 'server:r1CJmycQW'

app.destroy()

setTimeout(() => { }, 10000)
