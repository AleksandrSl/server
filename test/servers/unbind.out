
 INFO   Shutting down Logux server at 1970-01-01 00:00:00
